server/render/test/index.test.js

```diff
@@ -0,0 +1,232 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { shouldServerSideRender, setShouldServerSideRender } from '..';
+import { setShouldServerSideRenderLogin } from '../../../client/login/ssr';
+
+let mockReturnValues = {};
+
+jest.mock( 'lib/i18n-utils', () => {
+	return {
+		isDefaultLocale: () => mockReturnValues.isDefaultLocale,
+	};
+} );
+
+jest.mock( 'state/ui/selectors', () => {
+	return {
+		isSectionIsomorphic: () => mockReturnValues.isSectionIsomorphic,
+	};
+} );
+
+jest.mock( 'config', () => {
+	const fn = () => {};
+	fn.isEnabled = feature_key =>
+		feature_key === 'server-side-rendering' ? mockReturnValues.configServerSideRender : false;
+	return fn;
+} );
+
+const aSsrCompatibleContext = {
```

Comment:
I'd ditch the `a` article :smile: (partly because it sounds wrong, would need `an` before `S` since that's pronounced 'es'). Same below.

Reply:
a is now gone. An alternative could have been `some`. My intention here was to clarify that it's not THE context, but rather some random context.